features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/AdminTable/AdminTableBody.jsx

```diff
@@ -34,11 +44,16 @@ import { useTableContext } from './AdminTableContext';
  * @returns
  */
 function descendingComparator(a, b, orderBy) {
-    if (b[orderBy] < a[orderBy]) {
-        return -1;
-    }
-    if (b[orderBy] > a[orderBy]) {
-        return 1;
+    if (orderBy === 'role') {
+        const roleA = extractRoleName(a, orderBy);
```

Comment:
In the method definition, you accepting only 1 argument, here passing 2 args? , This `descendingComparator ` meant to be a generic comparator given the `orderBy ` value (column name), But here we only use it for role comparison ?

Reply:
Removed extra args with c198645.

The only sortable column in the Scope Mappings table is the `Roles` column. Earlier the comparator was looking for `orderBy` attribute value in the args passed (a and b) to this function but they were missing. Here, I've updated it to check whether orderBy is `role` and if so,  extract the correct role name from the args and compare them.